crates/codegen/parser_templates/src/rust/cst_visitor.rs

```diff
@@ -0,0 +1,130 @@
+use std::{ops::Range, rc::Rc};
+
+use super::cst::*;
+use super::kinds::*;
+
+#[allow(unused_variables)]
+pub trait Visitor<E> {
```

Comment:
are we not including visitors in the TypeScript output? Is this a new limitation?

Reply:
Not a new limitation, just not yet done. Visitors are so easy in TS to roll your own

Reply:
i.e. was never in the codebase

Reply:
Thanks for pointing out. I guess we can put this aside for later. Created #334 to track this.